packages/components/src/components/hds/stepper/list/step.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { guidFor } from '@ember/object/internals';
+import { modifier } from 'ember-modifier';
+
+import {
+  HdsStepperStatusesValues,
+  HdsStepperTitleTagValues,
+} from '../types.ts';
+import {
+  type HdsStepperStatuses,
+  type HdsStepperTitleTags,
+  type HdsStepperListStepIds,
+  HdsStepperStatusToSrOnlyText,
+} from '../types.ts';
```

Comment:
@dchyun Should the last one go above? And then you could import the types together without repeating "type".

Like:
```import {
  HdsStepperStatusesValues,
  HdsStepperTitleTagValues,
  HdsStepperStatusToSrOnlyText,
} from '../types.ts';

import type {
  HdsStepperStatuses,
  HdsStepperTitleTags,
  HdsStepperListStepIds,
} from '../types.ts';